test/ParseableInterface/optimization-level.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -typecheck -emit-parseable-module-interface-path %t/Lib.swiftinterface %s -O
+// RUN: %target-swift-frontend -build-module-from-parseable-interface %t/Lib.swiftinterface -Xllvm -sil-print-pass-name -o /dev/null 2>&1 | %FileCheck --check-prefix OPT %s
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend -typecheck -emit-parseable-module-interface-path %t/Lib.swiftinterface %s -Onone
+// RUN: %target-swift-frontend -build-module-from-parseable-interface %t/Lib.swiftinterface -Xllvm -sil-print-pass-name -o /dev/null 2>&1 | %FileCheck --check-prefix UNOPT %s
```

Comment:
Do -Xllvm options work outside of NDEBUG? I don't remember. Maybe add "REQUIRES: asserts" just because?

Another option would be to dump the SIL, but that's kind of fragile too.

Reply:
It seems there are a bunch of SIL/IRGen tests that pass `-Xllvm sil-*` arguments without any `REQUIRES`es. I think we're good, unless you think testing for `EagerSpecializer` is the wrong approach...

Reply:
It's not a bit implementation-detail-y. Maybe just leave a comment to say what you're really trying to test?